### PR TITLE
WRR-1116: Updated package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Update package dependencies to fix security vulnerabilities

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Update package dependencies to fix security vulnerabilities

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Updated node engine version for electron to `>=20.16.0`.

Major versions will be updated in another ticket:
electron  ^31.4.0  →  ^32.0.2
@types/node  ^20.16.5  →  ^22.5.4

### Links
[//]: # (Related issues, references)
WRR-1116

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

File: packages/theme/template/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "ilib": "^14.19.0 || ^14.19.0-webos1"
```
Comment (daniel-stoian-lgp) [inline packages/theme/template/package.json]: please update this  ilib as well

File: packages/webostv/template/package.json [hunk 2]
```diff
@@ -43,9 +43,9 @@
     "prop-types": "^15.8.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "web-vitals": "^4.2.1"
+    "web-vitals": "^4.2.3"
```
Comment (daniel-stoian-lgp) [inline packages/webostv/template/package.json:46]: please update ilib dependency from above

File: packages/agate/template/package.json [hunk 3]
```diff
@@ -39,9 +39,9 @@
     "prop-types": "^15.8.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "web-vitals": "^4.2.1"
+    "web-vitals": "^4.2.3"
```
Comment (daniel-stoian-lgp) [inline packages/agate/template/package.json:42]: please alsp update ilib to ^14.20.0

**Review by daniel-stoian-lgp — APPROVED** [review]

File: packages/electron/template/package.json [hunk 4]
```diff
@@ -25,7 +25,7 @@
   },
   "license": "UNLICENSED",
   "engines": {
-    "node": ">=20.14.0"
+    "node": ">=20.16.0"
```
Comment (hong6316) [inline packages/electron/template/package.json]: In PR description, there's no information about updating node version.
Please add it on `Additional considerations`.

Comment (hong6316) [reply packages/electron/template/package.json]: And, the node version need to be discussed about it.
Please search more info...
https://github.com/enactjs/templates/pull/97#discussion_r1764657353

Comment (paul-beldean-lgp) [reply packages/electron/template/package.json]: I left a comment in the other PR why I think v20 is right.
https://github.com/enactjs/templates/pull/97#discussion_r1785999405

**Pull request merged**